lib/memoizable/memory.rb

```diff
@@ -7,13 +7,13 @@ class Memory
 
     # Initialize the memory storage for memoized methods
     #
-    # @param [ThreadSafe::Cache] memory
+    # @param [Hash] memory
```

Comment:
__TODO:__ Extract a commit to remove this prior to adding it back as a `Hash`, since the original code did not accept an argument.